elasticsearch/1.9/README.md

```diff
@@ -0,0 +1,99 @@
+# How to use Elasticsearch on DC/OS
+
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a distributed, RESTful search and analytics engine.
+The [DC/OS Elasticsearch](https://github.com/mesos/elasticsearch) service is a Mesos framework that allows you to manage
+and use Elasticsearch in a flexible and scalable way.
+
+- Estimated time for completion: 5 minutes
+- Target audience: Data engineers and appops that want an search engine.
+- Scope: Install and use Elasticsearch.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Elasticsearch](#install-elasticsearch)
+- [Use Elasticsearch](#use-elasticsearch)
+- [Uninstall Elasticsearch](#uninstall-elasticsearch)
+
+## Prerequisites
+
+- A running DC/OS 1.9 cluster with 3 agents with each 1 CPU, 3 GB of RAM and 1 GB of disk space available.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed.
+
+## Install Elasticsearch
+
+To install Elasticsearch, do:
+
+```bash
+$ dcos package install elasticsearch
+The ElasticSearch DCOS Service implementation is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.
```

Comment:
The output I got here was a bit different, can you double-check yours?

This DC/OS Service is currently in preview. The ElasticSearch DCOS Service implementation is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.